wikidatarefisland/data_model/statement_filterer.py

```diff
@@ -0,0 +1,8 @@
+class StatementFilterer:
+    def __init__(self, filters):
+        self._filters = filters
+
+    def filter_statements(self, statements):
+        for statement_filter in self._filters:
+            statements = filter(statement_filter, statements)
```

Comment:
I'm not sure if it might make more sense to return a more flexible iterator, like List, what do you think?